Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/logs.xhtml

```diff
@@ -1,3 +1,5 @@
+C:\CONNECT\Product\Production\Adapters\General\CONNECTAdminGUI\src\main\webapp\logs.xhtml
+
```

Comment:
Out of scope for this ticket.  Agree this file should be removed but let's create a separate ticket to confirm.